docs/reference/config-options.md

```diff
@@ -284,7 +298,7 @@ federator:
 
 Some features (as of the time of writing this: only
 `conferenceCalling`) allow to set defaults for personal accounts in
-brig.  Those are taken into account in galley's end-points `GET
+brig. Those are taken into account in galley's end-points `GET
```

Comment:
https://www.theatlantic.com/science/archive/2018/05/two-spaces-after-a-period/559304/  (I'm not saying change it back, we're not doing this consistently.  But please don't make these changes in the future if you can avoid them.)

Reply:
understood